src_c/_pygame.h

```diff
@@ -80,11 +80,11 @@ typedef enum {
 typedef enum {
     SDL_NOEVENT = 0,
     /* SDL 1.2 allowed for 8 user defined events. */
-    SDL_NUMEVENTS = SDL_USEREVENT + (32768/4),
-    SDL_ACTIVEEVENT = SDL_NUMEVENTS,
-    PGE_EVENTBEGIN = SDL_NUMEVENTS,
+    SDL_ACTIVEEVENT = SDL_USEREVENT + 0x2000,
+    PGE_EVENTBEGIN = SDL_ACTIVEEVENT,
     SDL_VIDEORESIZE,
     SDL_VIDEOEXPOSE,
+    SDL_NUMEVENTS,
```

Comment:
I'm not sure what SDL_NUMEVENTS is correct here.

It should be the maximum number an event can be assigned.
See: http://sdl.beuc.net/sdl.wiki/SDL_UserEvent
> type is a value from SDL_USEREVENT to SDL_NUMEVENTS-1 

SDL_ACTIVEEVENT is an event that isn't in SDL2, so we use one of the user events for it.